### PR TITLE
CTCTRADERS-1641: Add non EN names when EN is unavailable

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/data/transform/TransformationImplicits.scala [hunk 1]
```diff
@@ -71,9 +71,10 @@ trait TransformationImplicits {
           val englishDetails: Option[JsValue] = array.find(
             x => (x \ "languageCode").as[String].toLowerCase == "en"
           )
+
           englishDetails match {
             case Some(value: JsObject) => Reads.pure(value)
-            case _                     => Reads.failed("Could not find element in array matching `en` language in path #/customsOfficeDetails")
+            case _                     => Reads.pure(array.head.as[JsObject])
```
Comment (ashwinirudrappa) [inline app/data/transform/TransformationImplicits.scala:77]: @MatthewAlexanderBeck What is the possibility of getting an empty **array**, in that case, array.head will fail how about using headOption.

**Review by ashwinirudrappa — COMMENTED** [review]
Can we add one more Test case to check how the system behaves when  we get an empty array.

File: app/data/transform/TransformationImplicits.scala (hunk 1 above)
Comment (MatthewAlexanderBeck) [reply app/data/transform/TransformationImplicits.scala:77]: Good shout, will add that in and return a null if none

**Review by ashwinirudrappa — APPROVED** [review]
